### ISSUE TITLE
Check test error on the test set

### ISSUE THREAD

**Issue by jonathanwangg** [jonathanwangg/Vancouver-Airbnb#4]
take combined training and validation set and have my best model be trained on that, then test it using the test set once (check if the test error and validation error agree)
